modular_skyrat/modules/cortical_borer/code/cortical_borer.dm

```diff
@@ -0,0 +1,605 @@
+GLOBAL_LIST_EMPTY(cortical_borers)
+
+/mob/proc/has_borer()
+	for(var/check_content in contents)
+		if(iscorticalborer(check_content))
+			return check_content
+	return FALSE
+
+/obj/machinery/door/Bumped(atom/movable/AM)
+	if(iscorticalborer(AM) && density)
+		if(!do_after(AM, 5 SECONDS, src))
+			return ..()
+		AM.forceMove(get_turf(src))
+		to_chat(AM, span_notice("You squeeze through [src]."))
+		return
+	return ..()
+
+/obj/item/organ/brain/Remove(mob/living/carbon/C, special = 0, no_id_transfer = FALSE)
+	. = ..()
+	var/mob/living/simple_animal/cortical_borer/cb_inside = C.has_borer()
+	if(cb_inside)
+		cb_inside.leave_host()
+
+/mob/living/simple_animal/cortical_borer
+	name = "cortical borer"
+	desc = "A slimy creature that is known to go into the ear canal of unsuspecting victims."
+	icon = 'modular_skyrat/modules/cortical_borer/icons/animal.dmi'
+	icon_state = "brainslug"
+	icon_living = "brainslug"
+	icon_dead = "brainslug_dead"
+	maxHealth = 50
+	health = 50
+	//they need to be able to pass tables and mobs
+	pass_flags = PASSTABLE | PASSMOB
+	//they are below mobs, or below tables
+	layer = BELOW_MOB_LAYER
+	//corticals are tiny
+	mob_size = MOB_SIZE_TINY
+	//because they are small, why can't they be held?
+	can_be_held = TRUE
+	///what chemicals borers know, starting with none
+	var/list/known_chemicals = list()
+	///what chemicals the borer can learn
+	var/list/potential_chemicals = list(/datum/reagent/medicine/spaceacillin,
+										/datum/reagent/medicine/potass_iodide,
+										/datum/reagent/medicine/diphenhydramine,
+										/datum/reagent/medicine/epinephrine,
+										/datum/reagent/medicine/antihol,
+										/datum/reagent/medicine/haloperidol,
+										/datum/reagent/consumable/nutriment,
+										/datum/reagent/consumable/hell_ramen,
+										/datum/reagent/consumable/tearjuice,
+										/datum/reagent/drug/thc,
+										/datum/reagent/drug/quaalude,
+										/datum/reagent/drug/happiness,
+										/datum/reagent/consumable/tea,
+										/datum/reagent/consumable/hot_coco,
+										/datum/reagent/toxin/formaldehyde,
+										/datum/reagent/impurity/libitoil,
+										/datum/reagent/impurity/mannitol,
+										/datum/reagent/medicine/c2/libital,
+										/datum/reagent/medicine/c2/lenturi,
+										/datum/reagent/medicine/c2/convermol,
+										/datum/reagent/medicine/c2/seiver,
+										/datum/reagent/lithium,
+										/datum/reagent/consumable/orangejuice,
+										/datum/reagent/consumable/tomatojuice,
+										/datum/reagent/consumable/limejuice,
+										/datum/reagent/consumable/carrotjuice,
+										/datum/reagent/consumable/milk,
+										/datum/reagent/medicine/salglu_solution,
+										/datum/reagent/medicine/mutadone,
+										/datum/reagent/toxin/heparin,
+										/datum/reagent/consumable/ethanol/beer,
+										/datum/reagent/medicine/mannitol,
+										/datum/reagent/drug/methamphetamine,
```

Comment:
I'm glad you included none of the hornychems but I think meth is too strong of a chemical to add to borers given its extremely long lasting consequences.

Reply:
makes sense.

Reply:
is there an alternative I could use?